spec/tasks/delete_items_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+require 'rake'
+describe "delete_items rake task " do 
+  before :all do
+    Rake.application.rake_require "tasks/sevendays"
+    Rake::Task.define_task(:environment)
+  end
+  it "deletes todos after seven days from creation day" do 
+    todo1 = Todo.create(user_id: 1, description: "this is a valid description", created_at: "2014-01-14 02:01:05")
+    expect(Todo.all.count).to eql(1)
+    expect(todo1.reload.persisted?).to eql(true)
+    Rake::Task['delete_items'].invoke
+    expect(Todo.all.count).to eql(0)
+    expect{todo1.reload.persisted?}.to raise_error(ActiveRecord::RecordNotFound)
+    
+  end
```

Comment:
The test is good, its testing what we wanted to, though it's got a little duplication of effort going on that is worth being aware of. Something like this would simplify things:

```ruby
it "deletes todos after seven days from creation day" do 

  create(:todo, description: "this is a valid description", created_at: 7.days.ago)

  expect(Todo.all.count).to eql(1)  
  Rake::Task['delete_items'].invoke
  expect(Todo.all.count).to eql(0)

end
```